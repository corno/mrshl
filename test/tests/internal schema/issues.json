[
    [ "unexpected property: 'a'", "error", 25, 5, 25, 8],
    [ "expected 'number' but found 'string' ", "error", 26, 10, 26, 13],
    [ "missing property: 'a1'", "error", 24, 1, 24, 2]
]