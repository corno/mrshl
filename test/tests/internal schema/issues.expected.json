[
    [ "deserialization", "warning", "expected a verbose type: ( )", 30, 1, 30, 2],
    [ "deserialization", "error", "unexpected property: 'a'. Choose from 'a1', 'b', 'c'", 31, 5, 31, 8],
    [ "deserialization", "error", "value 'C' must be unquoted", 33, 10, 33, 13]
]